src/components/core/Button.tsx

```diff
@@ -4,7 +4,7 @@ import { Button as RNPButton } from 'react-native-paper';
 import { ipctColors } from 'styles/index';
 
 interface IButtonProps {
-    modeType: 'green' | 'gray' | 'default';
+    modeType: 'green' | 'gray' | 'blue' | 'default';
```

Comment:
"default" is actually the "blue" (brand color). I suggest having just one.

Reply:
Reverted back already.